canvas_modules/common-canvas/src/common-properties/properties-controller.js

```diff
@@ -750,6 +750,15 @@ export default class PropertiesController {
 		this.propertiesStore.clearSelectedRows(propertyId);
 	}
 
+	/**
+	 * Disable table row move buttons for all propertyIds in given array
+	 * @param propertyIdArray Array of propertyIds
+	 *
+	 */
+	disableRowMoveButtons(propertyIdArray) {
```

Comment:
It looks like this is an array of strings.  Any reason you are using an array over a single property?  I'm guessing for a single property we'd have to allow for enabling it.  

Users should be passing in an object for the propertyId.  Similar to what we do with all other controller methods.  We can store it in redux as a string though.

Reply:
I am using an array, so host applications can add multiple propertyIds in array. And move row buttons will be disabled for all propertyIds. 
Example -` [{"name":"list_string"}, {"name":"list_string_error"}] `